sdk/scala-sdk/src/main/scala/kalix/scalasdk/action/Action.scala

```diff
@@ -153,14 +157,31 @@ object Action {
     }
   }
 }
+
 abstract class Action {
   @volatile
   private var _actionContext: Option[ActionContext] = None
 
+  /**
+   * An ExecutionContext to use when composing Futures inside Actions.
+   *
+   * Note that this ExecutionContext is only available when handling a message. It will throw an exception if accessed
+   * from constructor.
+   */
+  implicit final def executionContext: ExecutionContext = {
```

Comment:
I think this is a good (convenient) default and it's still possible to override if needed for some reason.